server/plugin_config.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/mattermost/mattermost-plugin-confluence/server/config"
+	"github.com/mattermost/mattermost-plugin-confluence/server/serializer"
+)
+
+var getPluginConfig = &Endpoint{
+	Path:            "/config",
+	Method:          http.MethodGet,
+	Execute:         handleGetPluginConfig,
+	IsAuthenticated: false,
```

Comment:
Why would we not have authentication on the getPluginConfig API?  At minimum a unauthenticated user could determine info about the server version.